web-registry/src/components/organisms/BuyCreditsModal/BuyCreditsModal.tsx

```diff
@@ -195,26 +193,18 @@ const BuyCreditsModal: React.FC<BuyCreditsModalProps> = ({
           validate={validationHandler}
           onSubmit={async (values, { setSubmitting }) => {
             setSubmitting(true);
-            if (location.pathname.includes('projects')) {
-              const track_data: any = {
-                price: selectedSellOrder?.askAmount,
-                batchDenom: selectedSellOrder?.batchDenom,
-                projectName: project.name,
-                creditClassName: project.id.split('-')[0],
-              };
-              if (values.retirementAction === 'manual') {
-                track_data['amountTradable'] = getFormattedNumber(
-                  values.creditCount,
-                );
-                track_data['amountRetired'] = getFormattedNumber(0);
-              } else if (values.retirementAction === 'autoretire') {
-                track_data['amountTradable'] = getFormattedNumber(
-                  values.creditCount,
-                );
-                track_data['amountRetired'] = getFormattedNumber(0);
-              }
-              track('buy2', track_data);
-            }
+            const trackData: {
+              price?: string;
+              batchDenom?: string;
+              projectName?: string | null;
+              creditClassName?: string;
+            } = {
+              price: selectedSellOrder?.askAmount,
+              batchDenom: selectedSellOrder?.batchDenom,
+              projectName: project.name,
+              creditClassName: project.id.split('-')[0],
```

Comment:
strictly speaking this is the on chain credit class **id**, not the credit class name

Reply:
i changed this `creditClassId`, i'm thinking it's ok and beneficial to keep "on chain" out of this name because i don't think that particular distinction is too important for those analyzing the analytics data.

e608b834

Reply:
agreed